ts/Core/Globals.ts

```diff
@@ -175,7 +208,7 @@ declare global {
 
 /**
  * Shared Highcharts properties.
- * @private
+ * @internal
```

Comment:
Because the Globals namespace is tagged with `internal`, `Highcharts.charts` is skipped. GlobalsBase is not tagged as internal - should be doclet be moved/copied there?